src/applications/disability-benefits/2346/components/ReviewCardField.jsx

```diff
@@ -284,8 +283,13 @@ class ReviewCardField extends React.Component {
       'vads-u-margin-top--1',
       'vads-u-width--auto',
     ].join(' ');
-
+    const { data } = this.props;
     const { street, city, country } = this.props.formData;
+    const copyOfTempAddress = data.temporaryAddress;
+    delete copyOfTempAddress['view:livesOnMilitaryBaseInfo'];
```

Comment:
you can do this with destructuring

```
const temp = { ['view:livesOnMilitaryBaseInfo]: removed, ...temporaryAddress }
const isTempAddressMissing = Object.values(temporaryAddress).every(
      prop => !prop,
    );
```
['view:livesOnMilitaryBaseInfo'] is being aliased into a variable called removed and everything else in the object is being spread into a variable called temporaryAddress

Reply:
Thanks, just updated.